modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/SystemObjectRelatedObjectEntriesTest.java

```diff
@@ -334,6 +335,22 @@ public void testGetOneToManySystemObjectRelatedObjectEntries()
 			},
 			Type.ONE_TO_MANY);
 
+		ObjectField titleObjectField = ObjectFieldTestUtil.addCustomObjectField(
+			TestPropsValues.getUserId(),
+			ObjectFieldConstants.BUSINESS_TYPE_TEXT,
+			ObjectFieldConstants.DB_TYPE_STRING, _userSystemObjectDefinition,
+			"a" + RandomTestUtil.randomString());
+
+		_userSystemObjectDefinition.setTitleObjectFieldId(
+			titleObjectField.getObjectFieldId());
+
+		_userSystemObjectDefinition =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_userSystemObjectDefinition);
```

Comment:
@devpedrotavares can you use the `_objectDefinitionLocalService.updateTitleObjectFieldId`?